platform/src/test/resources/test-read.yaml

```diff
@@ -0,0 +1,3 @@
+# Test Yaml File
+stellar:
+  test: test-value
```

Comment:
I don't see you testing this in `ConfiguratorTest`.

Reply:
Good catch. Since the purpose of `test-read.yaml` is to test if the configurator can read the file, the content of the file isn't important. I had emptied `test-read.yaml`.